src/components/MoneyRequestReportView/MoneyRequestReportTransactionsNavigation.tsx

```diff
@@ -12,25 +13,47 @@ import getEmptyArray from '@src/types/utils/getEmptyArray';
 
 type MoneyRequestReportRHPNavigationButtonsProps = {
     currentReportID: string;
+    parentReportID?: string;
+    policyID?: string;
 };
 
-function MoneyRequestReportTransactionsNavigation({currentReportID}: MoneyRequestReportRHPNavigationButtonsProps) {
+function MoneyRequestReportTransactionsNavigation({currentReportID, parentReportID, policyID}: MoneyRequestReportRHPNavigationButtonsProps) {
     const [reportIDsList = getEmptyArray<string>()] = useOnyx(ONYXKEYS.TRANSACTION_THREAD_NAVIGATION_REPORT_IDS, {
         canBeMissing: true,
     });
 
-    const {prevReportID, nextReportID} = useMemo(() => {
+    const [parentReportActions = new Map<string, string>()] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${parentReportID}`, {
```

Comment:
It would be great to add plain English comment explaining why we need this map, it is not something common in the app I would say